api/event.js

```diff
@@ -0,0 +1,88 @@
+const fetch = require('node-fetch')
+const gqlr = require('graphql-request')
+const { GraphQLClient } = gqlr
+const { API_URL } = require('../src/config')
+
+const EndPoint = `${API_URL}/graphql`
```

Comment:
Capitalised snake case for constants e.g. const END_POINT. GRAPHQL_ENDPOINT might be better though to make it clear

Reply:
The `EndPoint` followed the style of `GetParty`, which can be changed if the name convention can be confirmed above.

Reply:
Confirmed